support/logging/file.go

```diff
@@ -0,0 +1,122 @@
+//
+// Copyright (c) 2018 Cavium
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package logging
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/edgexfoundry/edgex-go/support/domain"
+)
+
+const (
+	rmFileSuffix string = ".tmp"
+)
+
+type fileLog struct {
+	filename string
+	out      io.WriteCloser
+}
+
+func (fl *fileLog) add(le support_domain.LogEntry) {
+	if fl.out == nil {
+		var err error
+		fl.out, err = os.OpenFile(fl.filename, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+		if err != nil {
+			//fmt.Println("Error opening log file: ", fl.filename, err)
```

Comment:
Remove comments

Reply:
All the comments are logs that I think are interesting to have, but I did not solved yet how to log them. It is one of the open issues.

My idea is to have a private function to generate log entries, to process them like the logs from other services. But is not done yet.